.github/workflows/publish_release.yml

```
@@ -8,9 +8,9 @@ on:
 
 jobs:
   tag_release:
-    runs-on: ubuntu-latest
+    runs-on: ${{inputs.runner}}
```

This should be reverted as this workflow doesn't accept any inputs

```suggestion
    runs-on: ubuntu-latest
```